### PR TITLE
add UserKeys table to xbrmm

### PR DESCRIPTION
**Pull request by om26er**

### CODE REVIEW

File: cfxdb/xbrmm.fbs [hunk 1]
```diff
@@ -259,3 +259,14 @@ table Transaction
...
 }
+
+table UserKey {
+    /// User key - a Ed25519 public key - for authenticating using WAMP-cryptosign.
+    pubkey: [uint8] (uint256);
+
+    /// Timestamp (epoch time in ns) of initial creation of this record.
+    created: uint64 (timestamp);
+
+    /// ID of user account this user key is owned by.
```
Comment (oberstet) [inline cfxdb/xbrmm.fbs:270]: // ID of XBR network member this user key is owned by (foreign key to [xbr.Member](https://github.com/crossbario/cfxdb/blob/20d6243f338ec30ae24bf989ceae137589e13e11/cfxdb/xbr.fbs#L100))

**Comment by oberstet** [comment]
last comment: I guess you have ensured to use new UUIDs for all tables (and indexes)?

otherwise: +1, looks good!

**Comment by om26er** [comment]
> last comment: I guess you have ensured to use new UUIDs for all tables (and indexes)?

Yes, otherwise CrossbarFX would eve start if there was a UUID conflict.

File: cfxdb/xbrmm.fbs (hunk 1 above)
Comment (om26er) [reply cfxdb/xbrmm.fbs]: Done

**Comment by om26er** [comment]
> so xbrmm.fbs change (which is fine!), and hence xbrmm.bfbs changes (right). but xbr.bfbs?

I just tried on master as well, it seems to recreate xbr.bfbs. Maybe its a leftover from a previous PR or something ? 
fbs

**Comment by oberstet** [comment]
> Yes, otherwise CrossbarFX would eve start if there was a UUID conflict.

yeah, so the built-in protection inside zlmdb worked (wasn't sure) :)

> I just tried on master as well, it seems to recreate xbr.bfbs. Maybe its a leftover from a previous PR or something ?

ok, fine! maybe also a subtle (insignificant) flatc change. guess it'll be fine and I should stop worrying;)

**Pull request merged**